etc/st2packgen/files/sensors/sensor_base.py

```diff
@@ -1,10 +1,16 @@
+# based on https://cobe.io/blog/posts/kubernetes-watch-python/
+
 import ast
 import json
 import sys
-
-import requests
-from requests.auth import HTTPBasicAuth
-
+import base64
+import select
+import socket
+import ssl
```

Comment:
My suggestion here is to add a dependency on `backports.ssl` so that this pack can still operate on various versions of Python using the available backports.